huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_topic.go

```diff
@@ -160,7 +161,7 @@ func resourceDmsRocketMQTopicCreate(ctx context.Context, d *schema.ResourceData,
 	return resourceDmsRocketMQTopicUpdate(ctx, d, meta)
 }
 
-func buildCreateRocketmqTopicBodyParams(d *schema.ResourceData, config *config.Config) map[string]interface{} {
+func buildCreateRocketmqTopicBodyParams(d *schema.ResourceData, _ *config.Config) map[string]interface{} {
```

Comment:
If the config is not used, we should remove it from the arguemnt list.

Reply:
done